src/encoded/genomic_data_service.py

```diff
@@ -114,6 +120,19 @@ def rna_get_request(self, format_='json'):
     def rna_get(self):
         self.rna_get_request()
 
+        for expression in self.expressions:
+            expression['featureID'] += f"/{expression['encodeID'].split('/')[-2]}"
+
+            for key in expression:
+                if expression[key] == None:
```

Comment:
Do you mean to do a falsy test (if Not expression[key]) or specific need to chek for None?

Reply:
Specific check for None. It comes back as None from the API. If we have a falsy test it might assign an empty string if the key is not present.

Reply:
I think `if expression[key] is None`?

Reply:
the behavior should be the same, but yes, `is None` is what should be used here